### DISCUSSION TITLE
Plugin that Menu swaps stair (SOLVED)

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15873] [Ideas]
The plugin would change EVERY climbable thing with: 
Left click to **climb-up**
Shift click to **climb-down**

This would make climbing up stuff way way more efficient and less annoying.

Currently I can swap left and shift click but literally every climbable is like custom made or have different ids which means I have to do it for every climbable thing in game when I come across it. Also sometimes I have to do it for every floor (ex. gnome stronghold)
![stairs](https://user-images.githubusercontent.com/73009020/199265611-8e34f2c3-2b5f-4222-9080-f197acdbdad6.png)

**Comment by Bram91** [comment]
This is already a thing and was added in [b3e87ee](https://github.com/runelite/runelite/commit/b3e87ee875b08ba977751d815bc54a9a23733fde)
It can be found in the `Menu Entry Swapper` plugin under the `Object Swaps` section
